deploy/helm/sumologic/values.yaml

```diff
@@ -2724,6 +2732,7 @@ otelcol:
             passthrough: false
             owner_lookup_enabled: true  # To enable fetching additional metadata using `owner` relationship
             extract:
+              delimiter: _
```

Comment:
Is this related to "enable sending queue and improve overall metrics performance due to e2e test" ? 😬

Reply:
this is `e2e` part :trollface: